native/c/zowex.cpp

```diff
@@ -573,10 +576,16 @@ int handle_console_issue(const ParseResult &result)
   ZCN zcn = {0};
 
   string console_name = result.find_kw_arg_string("console-name");
-  cout << "console name " << console_name << endl;
+  int timeout = result.find_kw_arg_int("timeout");
+
   string command = result.find_pos_arg_string("command");
   bool wait = result.find_kw_arg_bool("wait");
 
+  if (timeout > -1)
```

Comment:
Is a timeout of 0 going to be useful here? Couldn't this cause a command to timeout instantly, even before its finished running?